package.json

```diff
@@ -21,7 +21,7 @@
     "ember-ajax": "2.5.6",
     "ember-can": "0.8.3",
     "ember-cli": "2.12.0",
-    "ember-cli-app-version": "^2.0.1",
+    "ember-cli-app-version": "^3.0.0",
```

Comment:
Thanks, Jon. I am not even sure how it was changed. I committed my code, then I updated my fork, and the new JSON file was added to my local, then I had to to push it to my fork.  I am not sure where the JSON update came from, but sounds like it. I will close this PR and see if I can start over.